server/graphql/queries.js

```diff
@@ -540,14 +540,28 @@ const queries = {
       if (args.tags) query.where.tags = { [Op.overlap]: args.tags };
 
       if (args.orderBy === 'balance' && (args.ParentCollectiveId || args.HostCollectiveId || args.tags)) {
-        return rawQueries.getCollectivesWithBalance(query.where, args);
+        const queryResult = await rawQueries.getCollectivesWithBalance(query.where, args);
+        console.log(queryResult);
```

Comment:
Can we remove the `console.log`?

Reply:
Done in another PR! Thanks for this and the limit/offset catch!